mlkem/poly.c

```diff
@@ -305,16 +311,17 @@ void poly_frombytes(poly *r, const uint8_t a[MLKEM_POLYBYTES])
 
 void poly_frommsg(poly *r, const uint8_t msg[MLKEM_INDCPA_MSGBYTES])
 {
+  int i, j;
 #if (MLKEM_INDCPA_MSGBYTES != MLKEM_N / 8)
 #error "MLKEM_INDCPA_MSGBYTES must be equal to MLKEM_N/8 bytes!"
 #endif
 
-  for (int i = 0; i < MLKEM_N / 8; i++)
+  for (i = 0; i < MLKEM_N / 8; i++)
   __loop__(
     invariant(i >= 0 && i <= MLKEM_N / 8)
     invariant(array_bound(r->coeffs, 0, (8 * i - 1), 0, (MLKEM_Q - 1))))
   {
-    for (int j = 0; j < 8; j++)
+    for (j = 0; j < 8; j++)
```

Comment:
Can we keep the declaration at the top of the _block_?

Reply:
yes - I changed this now. Can you please double check as it's easy to miss.